homework15/Shapiro Andrey HT 15/Shapiro Andrei.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+app = FastAPI()
+title = "Books Storage"
```

Comment:
where do you use them?

should not it be something like:

`FastAPI( title = "Books Storage" )`